mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/mule/tasks/PostprocessMuleApplication.java

```diff
@@ -49,7 +50,7 @@ public String getFrom() {
 
   @Override
   public List<MigrationStep> getSteps() {
-    return newArrayList(new AttributesToInboundPropertiesScriptGenerator());
+    return newArrayList(new AttributesToInboundPropertiesScriptGenerator(), new MigrateApiConfigFlows());
```

Comment:
it must be a post migration step because during the API Kit migration the ApplicationModel is not yet saved. Also the UriParam transformation must be the first MuleEvent in a flow for the metadata to work.

Reply:
That's fine. What I meant is that it should be named as a post processor specific to Apikit and its code should live with the other Apikit migration components. Then it can be added in the correct order in the method I referenced above.